<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="WidgetCoordinates" name="widgetCoordinates" native="true"/>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="WidgetCNP" name="cnp" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>426</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPreferencias">
    <property name="title">
     <string>Preferencias</string>
    </property>
    <addaction name="actionOpciones_avanzadas"/>
   </widget>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo_proyecto"/>
    <addaction name="actionAbrir_proyecto"/>
    <addaction name="actionGuardar_proyecto"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuCoordenadas">
    <property name="title">
     <string>Coordenadas</string>
    </property>
    <addaction name="actionSelect_coordenadas"/>
    <addaction name="actionLimpiar_coordenadas"/>
    <addaction name="actionAyuda_coordenadas"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuCoordenadas"/>
   <addaction name="menuPreferencias"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpciones_avanzadas">
   <property name="text">
    <string>Opciones avanzadas</string>
   </property>
  </action>
  <action name="actionNuevo_proyecto">
   <property name="text">
    <string>Nuevo proyecto</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>&amp;Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionSelect_coordenadas">
   <property name="text">
    <string>Seleccionar archivo</string>
   </property>
  </action>
  <action name="actionLimpiar_coordenadas">
   <property name="text">
    <string>Limpiar coordenadas</string>
   </property>
  </action>
  <action name="actionAyuda_coordenadas">
   <property name="text">
    <string>Ayuda</string>
   </property>
  </action>
  <action name="actionAbrir_proyecto">
   <property name="text">
    <string>Abrir proyecto</string>
   </property>
  </action>
  <action name="actionGuardar_proyecto">
   <property name="text">
    <string>Guardar proyecto</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WidgetCoordinates</class>
   <extends>QWidget</extends>
   <header>../WidgetCoordinatesTP/widgetcoordinates.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetCNP</class>
   <extends>QWidget</extends>
   <header location="global">../WidgetRegistroTP/widgetcnp.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
